backend/src/main/java/backend/utils/SynonymsUtils.java

```diff
@@ -0,0 +1,39 @@
+package backend.utils;
+
+import backend.dto.synonyms.SynonymDto;
+import backend.exception.bad_request.BadSynonymRequest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SynonymsUtils {
+
+    public static List<String> getSynonyms(String baseWord) throws BadSynonymRequest {
+
+        RestTemplate restTemplate = new RestTemplate();
```

Comment:
Rest Template jest bodajże beanem w naszej apce i nie trzeba to tworzyć można dać autowired ale wtedy metoda nie może być statyczna